README.md

```diff
@@ -235,18 +235,17 @@ EOF
 )
 
 ## make sure the queue policy is valid JSON
-$ echo "$QUEUE_POLICY" | jq .
+echo "$QUEUE_POLICY" | jq .
 
 ## Save queue attributes to a temp file
-$ cat << EOF > /tmp/queue-attributes.json
+cat << EOF > /tmp/queue-attributes.json
 {
   "MessageRetentionPeriod": "300",
   "Policy": "$(echo $QUEUE_POLICY | sed 's/\"/\\"/g' | tr -d -s '\n' " ")",
-  "SqsManagedSseEnabled": true
```

Comment:
Please keep this attribute. Adding quotes around the value (i.e. `"true"`) will fix the error.